client/components/upgrades/gsuite/gsuite-dialog/product-details.jsx

```diff
@@ -34,11 +34,11 @@ class GoogleAppsProductDetails extends Component {
 		const { translate } = this.props;
 
 		return (
-			<div className="google-apps-dialog__product-details">
-				<div className="google-apps-dialog__product-intro">
-					<div className="google-apps-dialog__product-name">
+			<div className="gsuite-dialog__product-details">
+				<div className="gsuite-dialog__product-intro">
+					<div className="gsuite-dialog__product-name">
```

Comment:
Same remark about renaming this component to `GSuite`.

Reply:
I needed to change the css to make the linter happy component comes next. Separate PR